libs/ledger-live-common/src/wallet-api/useDappLogic.ts

```diff
@@ -386,7 +388,8 @@ export function useDappLogic({
             await new Promise<void>((resolve, reject) =>
               uiHook["account.request"]({
                 currencies: [getCryptoCurrencyById(requestedCurrency.currency)],
-                onSuccess: () => {
+                onSuccess: (account) => {
```

Comment:
You can check the lint before pushing locally to be sure the CI will pass
https://github.com/LedgerHQ/ledger-live/actions/runs/9225374232/job/25382725232
```suggestion
                onSuccess: account => {
```